src/js/definitions/ssn.js

```diff
@@ -2,6 +2,11 @@ import { validateSSN } from '../validation';
 import SSNWidget from '../widgets/SSNWidget';
 import SSNReviewWidget from '../review/SSNWidget';
 
+const schema = {
+  type: 'string',
+  pattern: '^[0-9]{9}$'
+};
+
 const uiSchema = {
```

Comment:
It seems inconsistent to have some `uiSchema`s pass in their labels and some (like this one) do not. Does it make sense to have all of them always be functions and take their labels/titles as the first arg? 

It's also inconsistent that `schema` are objects but `uiSchema` are not, but I can't think of any good reason to make `schema` a function; if you want to define whether something is `required` or not you can't put it here for example.

Reply:
Yes I could imagine some `schema` being functions in the future if you want to specify things like required or `minLength` or something like that. Perhaps it makes sense to make everything a function, even if some things don't have configurable inputs, just for consistency?